clarity_ext/driverfile.py

```diff
@@ -136,8 +136,8 @@ def file_key(self, file_name):
     def artifact(self):
         artifacts = [shared_file for shared_file in self.extension.context.shared_files
                      if shared_file.name == self.shared_file_name]
-        assert len(artifacts) == 1, "shared file {} could not be found".format(
-            self.shared_file_name)
+        assert len(artifacts) == 1, "Shared file '{}' not exactly single".format(
```

Comment:
So this particular change is only fixing the error message. But I believe my thinking at the time was that I didn't know if there would ever be more than one item in the list. The logic I had come up with needed the first item in the list, and in such cases I think it's prudent to throw an error or at least assert.